portfolio/src/main/webapp/index.html

```diff
@@ -9,11 +9,49 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <!-- <p>This is my portfolio.</p> -->
+      <h3>General Infomation</h3>
+      <p>
+        Hi, My name is Xiaoyue Wang. I am an undergraduate studying at University of California 
+        San Diego (UCSD) and double majoring in Applied Mathematics and Data Science. 
+      </p>
+      <p>
+        Coding language: Python, Java, HTML, Javascript
```

Comment:
optional: turn this into a proper list with the [li tag](https://www.w3schools.com/tags/tag_li.asp)